stl/inc/algorithm

```diff
@@ -2847,6 +2847,40 @@ namespace ranges {
             _STL_INTERNAL_STATIC_ASSERT(sentinel_for<_Se2, _It2>);
             _STL_INTERNAL_STATIC_ASSERT(indirectly_swappable<_It1, _It2>);
 
+#if _USE_STD_VECTOR_ALGORITHMS
+            using _Elem1 = remove_reference_t<iter_reference_t<_It1>>;
+            using _Elem2 = remove_reference_t<iter_reference_t<_It2>>;
+            if constexpr (same_as<_Elem1, _Elem2> && _Is_trivially_swappable_v<_Elem1> && //

```

Comment:
what are the line comments doing at the end of each line here?

Reply:
They are preventing clang-format from formatting them like this:
```
if constexpr (
    same_as<_Elem1,
        _Elem2> && _Is_trivially_swappable_v<_Elem1> && contiguous_iterator<_It1> && _Sized_or_unreachable_sentinel_for<_Se1, _It1> && contiguous_iterator<_It2> && _Sized_or_unreachable_sentinel_for<_Se2, _It2>) {

```